src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisAjaxController.java

```diff
@@ -1199,6 +1202,45 @@ public AnalysisProvenanceResponse getProvenanceByFile(@PathVariable Long submiss
 
 	}
 
+	@RequestMapping(value = "/{submissionId}/analysis-details")
```

Comment:
Since this is all ajax stuff why don't you create an ajax controller to handle it?  Also JavaDoc